package.json

```diff
@@ -0,0 +1,183 @@
+    "enquirer": "^2.3.6",
+    "eventemitter2": "^6.4.5",
+    "executable": "^4.1.1",
+    "execa": "^4.1.0",
+    "extract-zip": "^2.0.1",
+    "extend": "^3.0.2",
+    "fd-slicer": "^1.1.0",
+    "figures": "^3.2.0",
+    "extsprintf": "^1.3.0",
+    "forever-agent": "^0.6.1",
+    "form-data": "^2.3.3",
+    "fs.realpath": "^1.0.0",
+    "fs-extra": "^9.1.0",
+    "get-stream": "^5.2.0",
+    "getos": "^3.2.1",
+    "getpass": "^0.1.7",
+    "glob": "^7.2.3",
+    "global-dirs": "^3.0.0",
+    "http-signature": "^1.3.6",
+    "graceful-fs": "^4.2.10",
+    "has-flag": "^4.0.0",
+    "ieee754": "^1.2.1",
+    "indent-string": "^4.0.0",
+    "human-signals": "^1.1.1",
+    "inflight": "^1.0.6",
+    "inherits": "^2.0.4",
+    "is-fullwidth-code-point": "^3.0.0",
+    "ini": "^2.0.0",
+    "is-ci": "^3.0.1",
+    "is-installed-globally": "^0.4.0",
+    "is-stream": "^2.0.1",
+    "isexe": "^2.0.0",
+    "jsonfile": "^6.1.0",
+    "is-path-inside": "^3.0.3",
+    "json-stringify-safe": "^5.0.1",
+    "is-typedarray": "^1.0.0",
+    "json-schema": "^0.4.0",
+    "is-unicode-supported": "^0.1.0",
+    "isstream": "^0.1.2",
+    "jsbn": "^0.1.1",
+    "listr2": "^3.14.0",
+    "lazy-ass": "^1.6.0",
+    "lodash.once": "^4.1.1",
+    "jsprim": "^2.0.2",
+    "lodash": "^4.17.21",
+    "log-symbols": "^4.1.0",
+    "lru-cache": "^6.0.0",
+    "log-update": "^4.0.0",
+    "mime-db": "^1.52.0",
+    "merge-stream": "^2.0.0",
+    "minimatch": "^3.1.2",
+    "mime-types": "^2.1.35",
+    "minimist": "^1.2.6",
+    "mimic-fn": "^2.1.0",
+    "ms": "^2.1.2",
+    "once": "^1.4.0",
+    "ospath": "^1.2.2",
+    "npm-run-path": "^4.0.1",
+    "p-map": "^4.0.0",
+    "path-is-absolute": "^1.0.1",
+    "onetime": "^5.1.2",
+    "pend": "^1.2.0",
+    "pretty-bytes": "^5.6.0",
+    "pify": "^2.3.0",
+    "path-key": "^3.1.1",
+    "psl": "^1.8.0",
+    "performance-now": "^2.1.0",
+    "proxy-from-env": "^1.0.0",
+    "qs": "^6.5.3",
+    "pump": "^3.0.0",
+    "request-progress": "^3.0.0",
+    "punycode": "^2.1.1",
+    "restore-cursor": "^3.1.0",
+    "rfdc": "^1.3.0",
+    "safe-buffer": "^5.2.1",
+    "safer-buffer": "^2.1.2",
+    "rxjs": "^7.5.5",
+    "rimraf": "^3.0.2",
+    "shebang-command": "^2.0.0",
+    "semver": "^7.3.7",
+    "signal-exit": "^3.0.7",
+    "sshpk": "^1.17.0",
+    "slice-ansi": "^3.0.0",
+    "strip-ansi": "^6.0.1",
+    "shebang-regex": "^3.0.0",
+    "string-width": "^4.2.3",
+    "throttleit": "^1.0.0",
+    "supports-color": "^8.1.1",
+    "strip-final-newline": "^2.0.0",
+    "tmp": "^0.2.1",
+    "through": "^2.3.8",
+    "tough-cookie": "^2.5.0",
+    "tweetnacl": "^0.14.5",
+    "tunnel-agent": "^0.6.0",
+    "tslib": "^2.4.0",
+    "wrap-ansi": "^7.0.0",
+    "type-fest": "^0.21.3",
+    "verror": "^1.10.0",
+    "which": "^2.0.2",
+    "untildify": "^4.0.0",
+    "uuid": "^8.3.2",
+    "universalify": "^2.0.0",
+    "wrappy": "^1.0.2",
+    "yauzl": "^2.10.0",
+    "yallist": "^4.0.0"
+  },
```

Comment:
why did you install all these dependencies? are they required for cypress?

Reply:
I've installed cypress version@3 according to the video and automatically created the dependencies, but I'm going to installed cypress according workshop instructions